src/providers/github.py

```diff
@@ -2,10 +2,27 @@
 
 from github import Github
 import requests
-
+import json
 from scanner import scan_diff
 from rules import read_rules
 
+
+# Github API: https://docs.github.com/en/rest/reference/repos#create-a-commit-comment
+def leave_comment_on_commit( token, commit_url, path, position, comment ):
+    org = commit_url.split('/')[3]
+    repo = commit_url.split('/')[4]
+    commit_hash = commit_url.split('/')[6]
+    api_url = 'https://api.github.com/repos/' + org + '/' + repo + '/commits/' + commit_hash + '/comments'
+    print(api_url)
+    payload = {'body': comment, 'path': path, 'position':position}
+    headers = {'user-agent': 'hcss', 'Accept': 'application/vnd.github.v3+json', 'Authorization': 'token ' + token }
+    r = requests.post(api_url, data=json.dumps(payload), headers=headers)
```

Comment:
instead of `data=json.dumps(payload)` you can use `json=payload` and the requests library will handle it accordingly

Reply:
```suggestion
    r = requests.post(api_url, json=payload, headers=headers)
```